src/main/java/com/sivalabs/springblog/post/web/controllers/PostController.java

```diff
@@ -0,0 +1,30 @@
+package com.sivalabs.springblog.post.web.controllers;
+
+import com.sivalabs.springblog.post.domain.PagedResult;
+import com.sivalabs.springblog.post.domain.Post;
+import com.sivalabs.springblog.post.domain.PostService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/posts")
+public class PostController {
```

Comment:
For spring-blog application we are not building REST API. We will render web UI using Thymeleaf templates.
So, this need to be updated accordingly.